src/clj/rems/middleware.clj

```diff
@@ -81,7 +83,8 @@
               context/*roles* (when context/*user*
                                 (set/union (roles/get-roles (getx-user-id))
                                            (dynamic-roles/get-roles (getx-user-id))))]
-      (handler request))))
+      (with-mdc {:roles (str/join " " (sort context/*roles*))}
```

Comment:
Roles are not logged?

Reply:
There are usually lots of them, so they are not logged in by default to avoid log clutter. If they are needed for debugging something, you can add them to logback config and thanks to the periodic scan, the config changes are applied without restarting.